gtsam/hybrid/HybridBayesNet.cpp

```diff
@@ -0,0 +1,13 @@
+//
+// Created by Fan Jiang on 1/24/22.
+//
+
+#include <gtsam/hybrid/HybridBayesNet.h>
+
+namespace gtsam {
+
+GaussianMixture::shared_ptr HybridBayesNet::atGaussian(size_t i) {
```

Comment:
should this not be an abstract conditional ?

Reply:
This is `atGaussian`, the `at()` does the abstract one